create-cluster.sh

```diff
@@ -47,6 +47,10 @@ for image in \
   envoyproxy/envoy:distroless-v1.29.2 \
   envoyproxy/gateway-dev:72c0cc7 \
   docker.io/envoyproxy/gateway:v1.0.0 \
+  docker.io/prom/prometheus:v2.48.1 \
+  docker.io/envoyproxy/gateway:v1.0.0 \
+  envoyproxy/gateway-dev:72c0cc7 \
+  envoyproxy/envoy:distroless-v1.29.2 \
```

Comment:
```suggestion
  docker.io/prom/prometheus:v2.48.1 \
```
Looks like the additional three lines are duplicates and we already had those.